tutorials/observing_examples.ipynb

```diff
@@ -0,0 +1,372 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from synphot import Observation\n",
+    "import numpy as np\n",
+    "\n",
+    "obs = Observation(photlam_hatp11, agile)\n",
+    "obs.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To get the expected count rate for a specific collecting area, use the <a href=\"https://synphot.readthedocs.io/en/latest/api/synphot.observation.Observation.html#synphot.observation.Observation.countrate\">countrate()</a> method: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "11852206.200075638 ct / s\n"
+     ]
+    }
+   ],
+   "source": [
+    "r = 3.5 / 2 * u.m  # radius of 3.5m ARC telescope at APO\n",
+    "area = np.pi * r.to(u.cm) ** 2\n",
+    "\n",
+    "cr = obs.countrate(area=area)\n",
+    "print(cr)"
```

Comment:
`11852206.200075638 ct / s` seems like a lot to me, but then again I am not familiar with this object. @bmorris3 should check if this number makes sense.

Reply:
Agreed

Reply:
This example needs to be tweaked to use ARCTIC instead of Agile for comparison with my observations. But it turns out this count rate is correct to within a factor of 2! (You get TONS of counts when observing the brightest exoplanet host stars with a 3.5 m telescope).

Reply:
Is factor of 2 good or bad? 😸  What is the empirical value from real observation? I am curious!

Reply:
I think the factor of two is pretty good, given that (1) we're ignoring the atmosphere, and (2) we're assuming a blackbody.